### ISSUE TITLE
DHT persistence tests fail with a running node.

### ISSUE THREAD

**Issue by paulhauner** [sigp/lighthouse#871]
## Description

I'm getting the following error when running `cargo test --all --release`:

```
---- service::tests::test_dht_persistence stdout ----
thread 'service::tests::test_dht_persistence' panicked at 'called `Result::unwrap()` on an `Err` value: Error(Libp2p(Msg("Discv5 service failed. Error: Os { code: 98, kind: AddrInUse, message: \"Address already in use\" }")), State { next_error: None, backtrace: InternalBacktrace { backtrace: None } })', src/libcore/result.rs:1188:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```

I also have a beacon node running in the background. I assume that it's using a port that the tests expect to have open? It would be nice if we could run a node and also pass tests.

**Comment by paulhauner** [comment]
@pawanjay176 I think you might be familiar with this test?

**Comment by AgeManning** [comment]
Yeah, @pawanjay176 looks like will have to use non-standard config. Need to change this guy:
https://github.com/sigp/lighthouse/blob/master/beacon_node/network/src/service/tests.rs#L67

**Comment by pawanjay176** [comment]
Sorry, saw this now. Will push a fix soon :)

**Comment by paulhauner** [comment]
Resolved by #874
